### ISSUE TITLE
Search not working in "closed" folders

### ISSUE THREAD

**Issue by hifi25nl** [bugy/script-server#325]
I have added folders in script-server menu
Search function is not working if folder content is not opened and visible

**Comment by bugy** [comment]
Hi @hifi25nl thanks for reporting!

**Comment by bugy** [comment]
Hi @hifi25nl unfortunately I couldn't reproduce it.
If I do search, folders are never hidden. And if I open a folder, it shows only filtered elements

All closed:
![image](https://user-images.githubusercontent.com/1275813/89733363-50268b00-da55-11ea-9f09-3625fc4de6d5.png)

g2 open:
![image](https://user-images.githubusercontent.com/1275813/89733370-59175c80-da55-11ea-9010-389a75ad6bb8.png)

**Comment by hifi25nl** [comment]
You are right, it would be nice if after a search the  folders involved are opened.

**Comment by bugy** [comment]
I thought about it, but what if after a search you still have too many
elements?

hifi25nl <notifications@github.com> schrieb am So., 9. Aug. 2020, 17:54:

**Comment by hifi25nl** [comment]
Maybe...
